### PR TITLE
Detail view

### PR DESCRIPTION
**Pull request by AngelaLee18**
MVC

View Controller to show details

DESCRIPTION

In this branch I created a navigation controller to show some details of a talk that the user chooses. 

EVIDENCE

https://user-images.githubusercontent.com/109097550/188468695-966948ba-68a4-4cfc-a76c-58dd3a1ce9ed.mov

### CODE REVIEW

File: Ted Talk/Ted Talk/Controller/DetailViewController.swift [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    
+    @IBOutlet weak var views: UILabel!
+    var talk: DetailModel?
+    
+    @IBOutlet weak var publishedDate: UILabel!
+    @IBOutlet weak var name: UILabel!
+    
+    @IBOutlet weak var descriptiond: UILabel!
+    @IBOutlet weak var tags: UILabel!
+    @IBOutlet weak var webView: WKWebView!
```
Comment (ezequielvelezglobant) [inline Ted Talk/Ted Talk/Controller/DetailViewController.swift]: please order this, first the outlets then the properties

**Review by ezequielvelezglobant — CHANGES_REQUESTED** [review]
image
Review the UI, some labels like title or tags are not multi-line, and in small screen the text is cropped.

File: Ted Talk/Ted Talk/Controller/DetailViewController.swift (hunk 1 above)
Comment (AngelaLee18) [reply Ted Talk/Ted Talk/Controller/DetailViewController.swift]: 48d9fe8ed59aa13e64b1b26354a68ddcaecc183e

File: Ted Talk/Ted Talk/Controller/DetailViewController.swift [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    
+    // MARK: - View Life Cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+        ShowDetail()
+        
+        let myURL = URL(string: talk?.url ?? "")
+        let myRequest = URLRequest(url: myURL!)
```
Comment (ezequielvelezglobant) [inline Ted Talk/Ted Talk/Controller/DetailViewController.swift]: do not use force unwrap!

**Review by ezequielvelezglobant — CHANGES_REQUESTED** [review]
Review de UI, the searchbar is not working. you have constraints problems

File: Ted Talk/Ted Talk/Controller/DetailViewController.swift (hunk 2 above)
Comment (AngelaLee18) [reply Ted Talk/Ted Talk/Controller/DetailViewController.swift]: fef5bfbb6d3af8b4fda3b5698bda622a3187f4e2

Comment (AngelaLee18) [reply Ted Talk/Ted Talk/Controller/DetailViewController.swift]: Corrected 
fef5bfbb6d3af8b4fda3b5698bda622a3187f4e2

**Review by ezequielvelezglobant — APPROVED** [review]

**Review by JMVarelaGlobant — APPROVED** [review]

**Pull request merged**